.gitignore

```diff
@@ -158,4 +158,4 @@ yarn.lock
 # build/
 out
 temp/
-tmp/
+tmp/
```

Comment:
can you please add back new lines at the  end of files? they are all marked with symbol ⛔